### PR TITLE
Ability to execute on-chain views

### PR DESCRIPTION
**Pull request by vishakh**
Using the `run_script_view` RPC, ConseilJS should now be able to get view invocation results.

### CODE REVIEW

**Review by anonymoussprocket — APPROVED** [review]

File: src/chain/tezos/TezosNodeWriter.ts [hunk 1]
```diff
@@ -957,4 +957,44 @@ export namespace TezosNodeWriter {
...
+     * @param {string} chain_id The chain id hash, not to be confused with chainid. Default is mainnet chain_id, NetXdQprcVkpaWU.
+     * @returns {Promise<any>} JSON-encoded response
+     */
+    export async function runView(
+        server: string,
+        chainid: string,
+        contract_address: string,
+        view_name: string,
+        view_input: any,
+        chain_id: string = 'NetXdQprcVkpaWU',
```
Comment (gdsoumya) [inline src/chain/tezos/TezosNodeWriter.ts:979]: the chainID var is there twice

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: src/chain/tezos/TezosNodeWriter.ts (hunk 1 above)
Comment (gdsoumya) [reply src/chain/tezos/TezosNodeWriter.ts:979]: also we probably don't need the chain id as I see in all other functions we always use chainid as "main" there is no other value being used even for gas estimation or sending transactions. tagging @anonymoussprocket to confirm that's the case

Comment (gdsoumya) [reply src/chain/tezos/TezosNodeWriter.ts:979]: need block number as a param

File: src/chain/tezos/TezosNodeWriter.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import FetchSelector from '../../utils/FetchSelector'
 const fetch = FetchSelector.fetch;
 
 import LogSelector from '../../utils/LoggerSelector';
+import {block} from "../../../test/reporting/tezos/TezosConseilClient.responses";
```
Comment (gdsoumya) [inline src/chain/tezos/TezosNodeWriter.ts:18]: is this import needed?

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: src/chain/tezos/TezosNodeWriter.ts (hunk 1 above)
Comment (vishakh) [reply src/chain/tezos/TezosNodeWriter.ts]: I preserved the two chain ID parameters to keep the new function consistent with the old ones. To change this, we should edit all functions in a separate PR. 

Block number has been added.

File: src/chain/tezos/TezosNodeWriter.ts (hunk 2 above)
Comment (vishakh) [reply src/chain/tezos/TezosNodeWriter.ts]: Removed.

**Review by anonymoussprocket — APPROVED** [review]

**Pull request merged**